File: PHP/Akademik.php
<?php
class Mahasiswa extends Akademik
{
    private $nim = '';
    private $prodi = '';
    private $fakultas = '';

    // Constructors
    public function __construct($nim = '', $prodi = '', $fakultas = '')
    {
        $this->nim = $nim;
        $this->prodi = $prodi;
        $this->fakultas = $fakultas;
    }

    // Getter NIM
    public function getNIM()
    {
        return $this->nim;
    }

    // Setter NIM
    public function setNIM($nim)
    {
        $this->nim = $nim;
    }

    // Getter Prodi
    public function getProdi()
    {
        return $this->prodi;
    }

    // Setter Prodi
    public function setProdi($prodi)
    {
        $this->prodi = $prodi;
    }

    // Getter Fakultas
    public function getFakultas()
    {
        return $this->fakultas;
    }

    // Setter Fakultas
    public function setFakultas($fakultas)
    {
        $this->fakultas = $fakultas;
    }
}

File: Python/Akademik.py
from Mahasiswa import Mahasiswa


class Akademik(Mahasiswa):
    def __init__(self, nim="", prodi="", fakultas="", kampus="", email="", telp=""):
        super().__init__(nim, prodi, fakultas)
        self.__kampus = kampus
        self.__email = email
        self.__telp = telp

    # Setter & Getter
    def setKampus(self, kampus):
        self.__kampus = kampus

    def getKampus(self):
        return self.__kampus

    def setEmail(self, email):
        self.__email = email

    def getEmail(self):
        return self.__email

    def setTelp(self, telp):
        self.__telp = telp

    def getTelp(self):
        return self.__telp
